src/frontend/components/Tour/Tour.scss

```diff
@@ -11,7 +11,7 @@
 
 // Add the tooltip class styling
 .heroic-tour-tooltip {
-  background: var(--neutral-02);
+  background: var(--tour-popup-background, var(--neutral-02));
```

Comment:
adding specific css variables for tour colors to make it easier to customize without breaking other parts of the app